### PR TITLE
add  fixed alloy-nav as per instruction

### PR DESCRIPTION
**Pull request by sarajul-haque**

### CODE REVIEW

File: marketing/src/sections/navigation/AlloyNav/AlloyNav.css [hunk 1]
```diff
@@ -56,75 +50,63 @@ hr {
...
+.product-menu-li {
     display: flex;
     gap: 50px;
 }
 
-h4 {
-    font-weight: bold;
-}
-
 .product-items span {
```
Comment (amezng) [inline marketing/src/sections/navigation/AlloyNav/AlloyNav.css:105]: Use `.alloy-nav` infront of the classname

File: marketing/src/sections/navigation/AlloyNav/AlloyNav.css [hunk 2]
```diff
@@ -28,23 +28,17 @@ nav.alloy-nav {
...
     color: grey;
 }
 
-hr {
-    color: #d5d3d3;
-    border: 1px solid;
-    list-style: none;
-}
-
 .alloy-nav span {
```
Comment (amezng) [inline marketing/src/sections/navigation/AlloyNav/AlloyNav.css:42]: Rename the file to `MetalNav` and all the classes to `metal-nav`

**Review by amezng — CHANGES_REQUESTED** [review]

File: marketing/src/sections/navigation/AlloyNav/AlloyNav.css (hunk 2 above)
Comment (sarajul-haque) [reply marketing/src/sections/navigation/AlloyNav/AlloyNav.css]: Add after rename the file to MetalNav and all the classes to metal-nav

**Review by amezng — APPROVED** [review]
LGTM

**Pull request merged**